utils/products.js

```diff
@@ -1,152 +1,372 @@
-      "https://github.com/ds-jhartmann",
-      "https://github.com/ds-mkanal",
-      "https://github.com/jzbmw"
+      "https://github.com/almadigabor",
+      "https://github.com/tomaszbarwicki",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Item Relationship Service Team",
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request managed-simple-data-exchanger Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
+  },
+  {
+    productName: "Policy Hub",
+    productDescription:
+      "The Policy Hub enables data providers, consumers and app providers to access a single-point-of-truth for Catena-X policies, attributes and templates for policy rules.",
+    githubRepo: ["https://github.com/eclipse-tractusx/policy-hub"],
+    committers: [
+      "https://github.com/evegufy",
+      "https://github.com/Phil91",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request Tractus-X Policy Hub Team",
+    hasBoard: false,
+    showVersion: true,
   },
-
   {
     productName: "Portal & Marketplaces",
-    productDescription: "The Portal facilitates the operations for dataspace members (companies), it includes functionalities such as registration, technical onboarding and marketplaces.",
+    productDescription:
+      "The Portal facilitates the operations for dataspace members (companies), it includes functionalities such as registration, technical onboarding and marketplaces.",
     githubRepo: [
       "https://github.com/eclipse-tractusx/portal",
       "https://github.com/eclipse-tractusx/portal-backend",
       "https://github.com/eclipse-tractusx/portal-frontend",
       "https://github.com/eclipse-tractusx/portal-frontend-registration",
       "https://github.com/eclipse-tractusx/portal-assets",
-      "https://github.com/eclipse-tractusx/portal-iam"
+      "https://github.com/eclipse-tractusx/portal-iam",
+      "https://github.com/eclipse-tractusx/portal-shared-components",
     ],
     committers: [
       "https://github.com/evegufy",
       "https://github.com/ntruchsess",
       "https://github.com/oyo",
-      "https://github.com/Phil91"
+      "https://github.com/Phil91",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request Portal and Marketplaces Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "Predictive Unit Real-Time Information Service",
+    productDescription:
+      "Predictive Unit Real-Time Information Service (PURIS) for Short Term Demand and Capacity Management",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/puris",
+    ],
+    committers: [
+      "https://github.com/tom-rm-meyer-ISST",
+      "https://github.com/mhellmeier",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Portal and Marketplaces Team",
+    mailTo: "tractusx-dev@eclipse.org?subject=Request puris Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "quality-dashboard",
```

Comment:
product not available

Reply:
I assume the [app-dashboard](https://github.com/eclipse-tractusx/app-dashboard) but that's not a product